### ISSUE TITLE
Update NAME table to remove full license text from binary font files

### ISSUE THREAD

**Issue by davelab6** [adobe-fonts/source-serif-pro#3]
The full text of the OFL is in the NAME table ID 13 of the binary font files distributed by Adobe from sourceforge.

I recommend replacing it with the following to reduce the filesize and improve web font latency: 

```
This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL
```

I also recommend making the copyright notice in NAME table ID 1 match the copyright notice in the txt files distributed with the fonts:

```
Copyright 2014 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'.
```

**Comment by kenlunde** [comment]
What about the name.ID=14 string? The Source {Sans,Serif,Code} Pro fonts currently use "http://www.adobe.com/type/legal.html" as the string, which is a URL that points to Adobe's standard (and non-open source) license, so wouldn't it be better to use "http://scripts.sil.org/OFL" instead?

**Comment by davelab6** [comment]
On 14 March 2015 at 21:12, Dr. Ken Lunde <notifications@github.com> wrote:

> What about the name.ID=14 string? The Source {Sans,Serif,Code} Pro fonts
> currently use "http://www.adobe.com/type/legal.html" as the string, which
> is a URL that points to Adobe's standard (and non-open source) license, so
> wouldn't it be better to use "http://scripts.sil.org/OFL" instead?
>
Agree that would be better

**Comment by kenlunde** [comment]
I would also advocate changing the name.ID=13 string to remove the URL, because it is available at the name.ID=14 string (whose official name is "License Info URL"):
~~~
This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at the License Info URL. This Font Software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the SIL Open Font License for the specific language, permissions and limitations governing your use of this Font Software.
~~~
Thoughts?

**Comment by davelab6** [comment]
Why not simply,

This Font Software is licensed under the SIL Open Font License,
Version 1.1. This Font Software is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
implied. See the SIL Open Font License for the specific language,
permissions and limitations governing your use of this Font Software.

**Comment by kenlunde** [comment]
Even better.

**Comment by pauldhunt** [comment]
I believe this all had to go through our legal department.

On Mar 14, 2015, at 8:23 AM, Dr. Ken Lunde <notifications@github.com<mailto:notifications@github.com>> wrote:


Even better.

-

**Comment by kenlunde** [comment]
The proposed modifications result in a more concise name.ID=13 string, and also correct the name.ID=14 string to reference to the correct license. In any case, I am bouncing these suggested changes off of our Legal for confirmation.

**Comment by kenlunde** [comment]
Adobe Legal just gave the proverbial "green light" for the proposed tweaks.

name.ID=0 (Copyright notice):
~~~
Copyright 2014, 2015 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font
Name 'Source'.
~~~
name.ID=13 (License Description):
~~~
This Font Software is licensed under the SIL Open Font License, Version 1.1. This Font Software
is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
either express or implied. See the SIL Open Font License for the specific language,
permissions and limitations governing your use of this Font Software.
~~~
name.ID=14 (License Info URL):
~~~
http://scripts.sil.org/OFL
~~~

**Comment by n7s** [comment]
Given that various font viewing apps conveniently tend to limit themselves to show (parts of) the copyright field and do not expose the URL fields to the user by default, I would recommend that any  Designer URL, Manufacturer URL and License URL (along with a mention of the FAQ) continue to stay visible in fields like Copyright and Description. The chosen license should also ideally be referenced in the copyright field: IOW with just the font file, if you simply see the copyright statement with no other information available, how can you can tell what you are allowed to do or not with the font software?

There's the OpenType spec and then there's the reality of how the metadata is actually shown and used by designers. 

The speed argument of removing the actual text of the license has to be weighted against the propagation of a font where clear authorship and licensing metadata is much easier to ignore. Are you sure you really want that? Including for the non-web-oriented formats? There are probably other ways of making size and speed gains that do not involve loosing the entire legal text. The licensed item should continue travelling with its license clearly attached and not require extra effort to be aware of its terms.

BTW, the OFL.txt header template has the expression "copied below" to encourage authors not to widen the gap between the copyright statement and the actual licensing terms too much, then recipients are more likely to have access to the full context of what is allowed and not have to rely just on a external reference.   (Think of offline scenarios as well).

**Comment by davelab6** [comment]
Since font binaries are not the preferred form of modification, and sources
have to be fetched from upstream, I think URLs are sufficient even for
'offline' cases. I don't think anyone in the world has a computer but no
internet access at all. Sure, they might need to wait a couple days to get
online, but that's normal for them.

**Comment by frankrolf** [comment]
This issue has been fixed in [release 2.0](https://github.com/adobe-fonts/source-serif-pro/releases/tag/2.000R).
